Add organisation related use cases

joshtyf opened the pull request:
Added organisation related use cases to the DG

joshtyf committed:
Add organisation related use cases

josiahkhoo commented on docs/DeveloperGuide.md:
```diff
@@ -471,7 +467,98 @@ MSS:
+3. QuickCache asks the user to type in the tags one by one.
+4. User types in the tags one by one.
+5. User signals that he is finished.
+5. QuickCache adds the tags to the flashcard.
```
I think this should be number 6

joshtyf replied:
fixed in the latest commit!

josiahkhoo commented on docs/DeveloperGuide.md:
```diff
@@ -471,7 +467,98 @@ MSS:
+
+* 5a. Some of the tags are new tags
+	* QuickCache stores and remembers the tags.
+	* 
```
Empty point

joshtyf replied:
fixed in the latest commit!

josiahkhoo reviewed (changes requested):
Looks good, just a few typos to update

joshtyf committed:
Fix minor documentation errors

This commit fixes errors mentioned in the pull request review for adding organisational related use cases.

josiahkhoo reviewed (approved):
Looks good, thanks Josh!
